src/views/ErrorHandler.vue

```diff
@@ -1,25 +1,52 @@
-<template></template>
+<template>
+    <div v-if="title" class="container pad-bottom">
+        <SmallPage>
+            <Loader
+                :title="title"
+                :state="state"
+                :message="message"
+                :mainAction="mainActionText"
+                :alternativeAction="alternativeActionText"
+                @main-action="mainActionHandler"
+                @alternative-action="alternativeActionHandler"
+                />
+        </SmallPage>
+    </div>
+</template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
 import { State } from 'vuex-class';
-import { Static } from '../lib/StaticStore';
+import staticStore, { Static } from '../lib/StaticStore';
 import { ParsedRpcRequest, ParsedSimpleRequest, RequestType } from '../lib/RequestTypes';
 import { Errors } from '@nimiq/keyguard-client';
 import { WalletStore } from '../lib/WalletStore';
 import KeyguardClient from '@nimiq/keyguard-client';
+import { SmallPage } from '@nimiq/vue-components';
+import Loader from '../components/Loader.vue';
+import { ERROR_ACCOUNT_REMOVED } from '../lib/Constants';
 
-@Component
+@Component({components: {SmallPage, Loader}})
 export default class ErrorHandler extends Vue {
     @Static protected request!: ParsedRpcRequest;
     @Static protected keyguardRequest?: KeyguardClient.Request;
     @State protected keyguardResult!: Error;
 
+    private title = '';
+    private state = Loader.State.ERROR;
+    private message = '';
+    private mainActionText = '';
+    private alternativeActionText = '';
+    private mainActionHandlerDict: {[error: string]: (event: Event) => void} = {};
+    private alternativeActionHandlerDict: {[error: string]: (event: Event) => void} = {};
```

Comment:
Instead of using the dictionaries here, couldn't you simply directly assign the handler to properties `mainActionHandler` and `alternativeActionHandler`?

Reply:
Yes, I think so. I had it differently first because I feared Vue's reactivity will break it. But then had to change it to regular dicts and it works. So now I can simplify it again :D